ca_profile.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Maintainer>
-  <Icon>https://raw.githubusercontent.com/manuel-rw/unraid-templates/master/ca_maintainer_avatar.jpeg</Icon>
+  <Icon>https://raw.githubusercontent.com/manuel-rw/unraid-templates/master/icons/ca_maintainer_avatar.jpeg</Icon>
```

Comment:
You can use github.com/manuel-rw.png as the url for your image. It’s a redirect by the GitHub api. Small tip :)

Reply:
Thanks, will update! 👍